src/components/InputText/styles.module.scss

```diff
@@ -0,0 +1,20 @@
+@import 'variables/colors';
+
+.label {
+  color: $black-main;
+  display: block;
```

Comment:
unnecessary, if the problem is about space fill, use flex in the container and make it direction to `column`